### DISCUSSION TITLE
Subscriptions missing when pulling data for MCA

### DISCUSSION THREAD

**Discussion by CaddyG** [microsoft/finops-toolkit#755] [Q&A] [answered]
Deployment was successful and PowerBi is pulling data in from the MCA, but only 4 of the 7 subscriptions are present. User has owner permission for all 7 subs. Looking for insight as to what is causing that.

**Comment by flanakin** [comment, marked as answer]
If anyone runs into this, there was a bad version of the CostSummary.pbix that was uploaded to the release. This has been fixed, so simply [re-download the CostSummary.pbix](https://aka.ms/ftk/latest) and you should be fine. Sorry for the inconvenience!
